imports/ui/unit-explorer/unit-explorer.jsx

```diff
@@ -12,62 +12,169 @@ import { setDrawerState } from '../general-actions'
-          <div className='br2 bg-white card-shadow-1 w-100 pt2 flex flex-column'>
-            <div className='ph2 flex flex-column'>
-              <h4 className='semi-dark-gray ma0 ml1 fw5 lh-title no-shrink'>ALL UNITS</h4>
-              <div className='overflow-auto pb2'>
-                {unitList.length === 0 ? (
-                  <div className='f6 i silver ba b--moon-gray mt2 pa2 tc br1'>
-                    You have no units managed with Unee-T yet
-                  </div>
-                ) : unitList.map(({ id, name, description, metaData }) => (
-                  <MenuItem key={id} innerDivStyle={{padding: 0}} onClick={() => dispatch(push(`/unit/${id}`))}>
-                    <div className='mt2 ba b--moon-gray br1 w-100 flex items-center pa2'>
-                      <FontIcon className='material-icons' color='var(--semi-dark-gray)'>home</FontIcon>
-                      <div className='ml3 mv1 semi-dark-gray lh-copy flex-grow overflow-hidden'>
-                        <div className='ti1 ellipsis'>{metaData.displayName || name}</div>
-                        <div className='flex'>
-                          <div className='ti1 ellipsis flex-grow'>{metaData.moreInfo || description}&nbsp;</div>
-                          <div
-                            onClick={evt => {
-                              evt.stopPropagation()
-                              dispatch(push(`/case/new?unit=${id}`))
-                            }}
-                            className='f6 ellipsis ml3 pl1 mv1 bondi-blue fw5 no-shrink'
-                          >
-                            Add case
-                          </div>
-                        </div>
-                      </div>
-                    </div>
-                  </MenuItem>
-                ))}
+        <RootAppBar title='My Units' onIconClick={() => dispatch(setDrawerState(true))} shadowless />
+        <div className='flex-grow flex flex-column overflow-hidden'>
+          <Tabs
+            className='no-shrink'
+            onChange={this.handleChange}
+            value={this.state.slideIndex}
+            inkBarStyle={{backgroundColor: 'white'}}
+          >
+            <Tab label='Active' value={0} />
+            <Tab label='Disabled' value={1} />
+          </Tabs>
+
+          <div className='flex-grow flex flex-column overflow-auto'>
+            <SwipeableViews
+              index={this.state.slideIndex}
+              onChangeIndex={this.handleChange}
+            >
+              {/* tab 1 */}
+              <div className='flex-grow bb b--very-light-gray bg-white'>
+                <FilteredUnitsList filteredUnits={activeUnits} dispatch={dispatch} currentUser={currentUser} />
+              </div>
+              {/* tab 2 */}
+              <div className='flex-grow bb b--very-light-gray bg-white'>
+                <FilteredUnitsList filteredUnits={disabledUnits} dispatch={dispatch} currentUser={currentUser} value={this.state.slideIndex} />
```

Comment:
Rename value to something more significant, like `showAddButtons`, and it shouldn't use the current slide index (it is only implicitly related to the value, and works by chance)... it should be either 'true' or 'false'